src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/error/AbstractExpectedError.groovy

```diff
@@ -0,0 +1,39 @@
+package org.openkilda.functionaltests.error
+
+import org.openkilda.messaging.error.MessageError
+import org.springframework.http.HttpStatus
+import org.springframework.web.client.HttpClientErrorException
+
+import java.util.regex.Pattern
+
+abstract class AbstractExpectedError extends HttpClientErrorException{
```

Comment:
Not sure why it extends HttpClientErrorException 🤔  It's not really an exception, is it? I mean, this is meant to help with verifications, not to be thrown

Reply:
It eliminates hundreds of IDE and build warnings about comparison of incompatible types. Doesn't look clear, but I consider this 'hack' as acceptable in this particular case.